docs/Authenticating-and-Authorizing-Users-via-keycloak.md

```diff
@@ -252,6 +253,85 @@ If all goes well, the following should happen:
 
 If this does not happen, see the Troubleshooting Tips  below.
 
+### Add client for OAuth2 token-based data access
+With cBioPortal instances that require user authentication the API can be queried when including a data access tokens in the request header (see [Authenticating Users via Tokens](Authenticating-Users-via-Tokens.md)). When using the `oauth2` implementation, KeyCloak can be configured as OAuth2 authentication provider that distributes data access tokens to users and validates these tokens when used while querying the API. This feature is enabled by creating a `cbioportal_api` OpenID Connect client that has access to the user roles defined in the `cbioportal` SAML client.
```

Comment:
Suggest rewording  in first sentence - s/tokens/token/
Suggest rewording in second sentence - "Keycloak can be configured as an OAuth2..."

Reply:
Updated